travvy/src/components/OnboardingPage.vue

```diff
@@ -0,0 +1,154 @@
+        <option>Prefer not to say</option>
+    </select>
+    <br>
+    <br>
+    <label>Please input your age: </label>
+    <input type="text" > 
+    <br>
+    </div>
+    <br>
+    <label>Please select the type of activity you enjoy most: </label>
+    <br>
+    <br>
+    <button v-on:click="reverseMessage" class="nextActivity">Family</button>   <button v-on:click="reverseMessage" class="nextActivity">Historical</button>   <button v-on:click="reverseMessage" class="nextActivity">Active</button>  <button v-on:click="reverseMessage" class="nextActivity">Nature</button>  <button v-on:click="reverseMessage" class="nextActivity"> Entertainment</button> <button v-on:click="reverseMessage" class="nextActivity">Food</button> 
+    <br>
+    <br>
+    <label>What is your dream vacation spot?: </label>
+    <input type="text"> 
+    <br>
+    <br>
+    <label>Please input your budget: </label>
+    <input type="text">
+    <br>
+    <br>
+    <label>Please include your Instagram handle!: </label>
+    <input type="text"> 
+    <br>
+    <br>
+    <!-- Button to go to the Home page --> 
+    <router-link to="/Home" tag="button" class="button"><span>Get Started!</span></router-link> 
+    <br>
+    
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Onboarding',
+  props: {
+    msg: String
+  }
+}
+</script>
+
+<!-- Add "scoped" attribute to limit CSS to this component only -->
+<style scoped>
+
+.button {
+  background-color: #FF5F00; 
+  border: none;
+  color: white;
+  padding: 15px 32px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+  border-radius: 12px;
+}
+
+.button span {
+  cursor: pointer;
+  display: inline-block;
+  position: relative;
+  transition: 0.5s;
+}
+
+.button span:after {
+  content: '\00bb';
+  position: absolute;
+  opacity: 0;
+  top: 0;
+  right: -20px;
+  transition: 0.5s;
+}
+
+.button:hover span {
+  padding-right: 25px;
+}
+
+.button:hover span:after {
+  opacity: 1;
+  right: 0;
+}
+
+
+h3 {
+  margin: 40px 0 0;
+}
+ul {
+  list-style-type: none;
+  padding: 0;
+}
+li {
+  display: inline-block;
+  margin: 0 10px;
+}
+a {
+  color: #42b983;
+}
+
+.nextActivity{
+  background-color: #89C4A0;
+  border: none;
+  color: white;
+  padding: 20px 35px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+  border-radius: 24px;
+}
+
+.nextActivity:hover {
+  background-color: #C7EEA9;
+}
+
+.container {
+  /* Add shadows to create the "card" effect */
+  background-color: #F0E4D8;
+  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+  margin: 0 400px;
+  transition: 0.3s;
+}
+
+/* On mouse-over, add a deeper shadow */
+.container:hover {
+  box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2);
+}
+
+/* Add some padding inside the card container */
+.container {
+  padding: 22px 10px;
+}
+
+.container {
+  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+  transition: 0.3s;
+  border-radius: 15px; /* 5px rounded corners */
+}
```

Comment:
You should be able to take this out, most if not all of it is in the app.vue file

Reply:
You're right, though a lot of it is currently specific to our onboarding styling! I'll be moving those over to the app.vue file as well so the styling is cohesive throughout the website. :))

Reply:
Great, thanks Saba!